streamz/dataframe/aggregations.py

```diff
@@ -202,10 +203,10 @@ def diff_loc(dfs, new, window=None):
     """
     dfs = deque(dfs)
     dfs.append(new)
-    mx = max(df.index.max() for df in dfs)
-    mn = mx - window
+    mx = pd.Timestamp(max(df.index.max() for df in dfs))
+    mn = pd.Timestamp(mx) - window
     old = []
-    while dfs[0].index.min() < mn:
+    while pd.Timestamp(dfs[0].index.min()) < mn:
         o = dfs[0].loc[:mn]
```

Comment:
I'm curious as to how casting to a pandas timestamp helps here.

Reply:
For now, cudf's `df.index.min()` (or max()) returns a `numpy.datetime64` as opposed to `pandas._libs.tslibs.timestamps.Timestamp`. Hence the explicit cast to make it compatible with Pandas Timedelta, which is required for these operations. 

The statements modified for this purpose would be redundant for Pandas, since the types are compatible with Pandas Timedelta.